codec/address/bech32_codec.go

```diff
@@ -1,32 +1,32 @@
-package keeper
+package address
```

Comment:
Putting the Bech32Codec has one small disadvantage: it will make it harder to refactor auth into a go.mod, since it will depend on the root sdk. But maybe it's fine for now, other go-modded sdk modules also depend on root.